services/app-api/forms/defaultMeasures.ts

```diff
@@ -193,6 +193,13 @@ export const defaultMeasureTemplates = {
           },
         ],
       },
+      {
+        type: ElementType.TextAreaField,
+        id: "additional-notes-field",
+        helperText:
+          "If applicable, add any notes or comments to provide context to the reported measure result",
+        label: "Additonal notes/comments (optional)",
```

Comment:
Small spelling mistake for all of them including the ones in `pomMeasures`.ts
`Additonal` -> `Additional`